README.md

```diff
@@ -1 +1,135 @@
-# math-magicians
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 Math Magicians React app <a name="about-project"></a>
+<br>
+
+
+<br>
+"Math magicians" is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to:<br>
+- Make simple calculations.<br>
+- Read a random math-related quote.<br>
+
+<br>
+<br>
+
+## 🛠 Built With <a name="built-with"></a>
+<br>
+
+
+  <summary>Page was built using React</summary>
+
+<br>
+
+### Key Features <a name="key-features"></a>
+
+<br>
+
+- **Project is created using React**
+- **Easu to reuse components for future projects**
```

Comment:
- [ ] [OPITONAL] As a developer, it's important to not only write clean and well-organized code but also to document your work effectively. One key aspect of this is creating a detailed and informative README file. In particular, the Key Features & Future Features section of your README presents a great opportunity to showcase your creativity and vision for the project. I would highly recommend adding a compelling and descriptive feature to this section to grab the attention of recruiters or future clients. 😊